### PR TITLE
Test optional `credentials.id` for `/credentials/issue` requests.

### LINKED ISSUE #521: Update Positive `credentials_issue` and `credentials_verify` tests to remove ids.
We want to use the minimal set of data required to pass these tests so we want to remove the credential `id`' from the credential used in each test.

Additionally, we want to add a second positive test for each of these that use the `id` as well so we have full coverage of what's allowed in the VC-API spec.

Additional context: https://github.com/w3c-ccg/traceability-interop/pull/519#pullrequestreview-1301471651

cc @brownoxford

### PR DESCRIPTION
**Pull request by brownoxford**
This PR removes the optional `credential.id` from the minimal request body used for testing `/credentials/issue`, and adds a new positive test to ensure that the endpoint works if `credential.id` is included in the request body.

Tests for `/credentials/verify` have not been changed, as verifiable credentials must have an ID.

Fixes #521

### CODE REVIEW

**Review by mkhraisha — APPROVED** [review]

**Review by OR13 — CHANGES_REQUESTED** [review]
This PR will break on GS1 VCs

File: tests/conformance_suite.postman_collection.json [hunk 1]
```diff
@@ -6797,6 +6797,81 @@
...
+											"",
+											"pm.test(\"response validates against schema\", function() {",
+											" const schemaString = pm.collectionVariables.get(\"responseSchema201CredentialsIssue\");",
+											" pm.response.to.have.jsonSchema(JSON.parse(schemaString));",
+											"});",
+											"",
+											"// Current OpenAPI v3.x schema cannot require specific array contents",
+											"pm.test(\"response @context contains traceability/v1\", function() {",
+											" const context = pm.response.json().verifiableCredential[\"@context\"];",
+											" pm.expect(context).to.contain(\"https://w3id.org/traceability/v1\");",
```
[the remark this answers is not in the document]
Comment (OR13) [reply tests/conformance_suite.postman_collection.json:6819]: Let us link to an issue to resolve this problem and I will approve.

**Comment by brownoxford** [comment]
Discussed on call - will create a new issue to track removal of this legacy part of the tests that reference required context content.

File: tests/conformance_suite.postman_collection.json [hunk 2]
```diff
@@ -6797,6 +6797,81 @@
...
+											"});",
+											"",
+											"pm.test(\"response verifiableCredential.id matches request credential.id\", function() {",
+											" const { id } = pm.response.json().verifiableCredential.id;",
+											" pm.expect(id).to.equal(pm.variables.get(\"id\"))",
```
Comment (OR13) [inline tests/conformance_suite.postman_collection.json:6824]: IMO this is the only test that should be present for this case... it is a mistake to repeat tests everywhere.

File: tests/conformance_suite.postman_collection.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (brownoxford) [reply tests/conformance_suite.postman_collection.json]: @OR13 I've removed the legacy test from this new portion of the suite, and created #526 to track removal of the remaining instances of this.

**Review by OR13 — APPROVED** [review]

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**